CRM/ACL/BAO/ACL.php

```diff
@@ -123,20 +118,16 @@ protected static function getACLs(int $contact_id) {
   protected static function getACLRoles($contact_id = NULL) {
     $contact_id = CRM_Utils_Type::escape($contact_id, 'Integer');
 
-    $rule = new CRM_ACL_BAO_ACL();
-
-    $contact = CRM_Contact_BAO_Contact::getTableName();
-
     $query = 'SELECT acl.* FROM civicrm_acl acl';
     $where = ['acl.entity_table = "civicrm_acl_role" AND acl.entity_id IN (' . implode(',', array_keys(CRM_Core_OptionGroup::values('acl_role'))) . ')'];
```

Comment:
@eileenmcnaughton and yeh nothing can go wrong with this right?

Reply:
@seamuslee001 It's kinda creepy. The code is sooo weird & this isn't even my first pass at cleanup on it - I did some quite some time back too